### PR TITLE
Adding r-jbrowser package

### PR DESCRIPTION
**Pull request by ctuni**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ctuni** [comment]
@cbrueffer Hi! I followed the instructions and think everything works. Let me know what else can I do :)

File: recipes/r-jbrowser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+about:
+  home: https://gmod.github.io/JBrowseR/ https://github.com/GMOD/JBrowseR
+  license: Apache-2.0
+  summary: Provides an R interface to the JBrowse 2 genome browser. Enables embedding a JB2 genome
+    browser in a Shiny app or R Markdown document. The browser can also be launched
+    from an interactive R console. The browser can be loaded with a variety of common
+    genomics data types, and can be used with a custom theme.
+  license_family: APACHE
+  license_file: LICENSE.txt
```
[the remark this answers is not in the document]
Comment (ctuni) [reply recipes/r-jbrowser/meta.yaml:69]: I did not know that! I'll change it now, thank you very much for the help :)

**Comment by ctuni** [comment]
Just seen that there is no LICENSE.md file in the CRAN tarball. Is this normal? Should I revert the changes?

**Comment by cbrueffer** [comment]
Argh, you're right, it's only in the git repo. All good then, thanks for reverting.

**Pull request merged**